src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -98,6 +98,15 @@ protected ActionResult execute() {
                                                         new Text(emailContent),
                                                         null);
             data.emailToEdit.emailId = emailId;
+        } else {
+            if (addressModeOn) {
+                statusToAdmin = "Email sent to " + addressReceiverListString;
+                statusToUser.add(new StatusMessage("Email sent to "
+                        + addressReceiverListString, StatusMessageColor.SUCCESS));
+            } else {
+                statusToAdmin = "Email sent to " + groupReceiverListFileKey;
+                statusToUser.add(new StatusMessage("Email sent to " + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
+            }
```

Comment:
@Neurrone how about this?
`if (addressModeOn && groupModeOn) {
                statusToAdmin = "Email sent to " + addressReceiverListString + "," + groupReceiverListFileKey;
                statusToUser.add(new StatusMessage("Email sent to "
                        + addressReceiverListString + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
            } else if (addressModeOn) {
                statusToAdmin = "Email sent to " + addressReceiverListString;
                statusToUser.add(new StatusMessage("Email sent to " + addressReceiverListString,
                        StatusMessageColor.SUCCESS));
            } else {
                statusToAdmin = "Email sent to " + groupReceiverListFileKey;
                statusToUser.add(new StatusMessage("Email sent to " + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
            }`
takes care of all the three cases.